src/fn_call.rs

```diff
@@ -693,4 +729,23 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
     fn write_null(&mut self, dest: PlaceTy<'tcx, Borrow>) -> EvalResult<'tcx> {
         self.eval_context_mut().write_scalar(Scalar::from_int(0, dest.layout.size), dest)
     }
+
+    // FIXME: unify with emulate_foreign_item if we ever end up having overlap
+    fn emulate_dlsym_item(
+        &mut self,
+        id: u32,
+        args: &[OpTy<'tcx, Borrow>],
+        dest: PlaceTy<'tcx, Borrow>,
+        ret: mir::BasicBlock,
+    ) -> EvalResult<'tcx> {
+        let this = self.eval_context_mut();
+        match id {
+            // epoll_create1
+            0 => {
```

Comment:
No magic constants please.  Seems like a C-like enum would be appropriate here, or so?